gtk3-macros/src/lib.rs

```diff
@@ -21,7 +21,7 @@ use syn::{parse_macro_input, DeriveInput};
 /// Specify that `MyWidget` is using a composite template and load the
 /// template file the `composite_template.ui` file.
 ///
-/// ```
+/// ```compile_fail
```

Comment:
Was it supposed to fail at compilation?

Reply:
It could've never compiled so I'd assume yes. It's also not tested by the CI.

Let me create an issue for that, thanks for the reminder.

Reply:
https://github.com/gtk-rs/gtk-rs/issues/468

Reply:
Note that the macros themselves are tested by the examples

Reply:
Not the problem here. ;)

I explained it a bit more on the issue you opened.